### DISCUSSION TITLE
What does the Compatibility mode 'properties.merging' mean?

### DISCUSSION THREAD

**Discussion by XIYU11864** [clean-css/clean-css-cli#92] [Q&A]
Acoding to the readme, it 'controls property merging based on understandability', but I don't get it and it is nothing differents between enable and disanle it. 
<img width="386" alt="image" src="https://github.com/clean-css/clean-css-cli/assets/75518225/43609317-ed8e-49b6-ad1b-4bcc109190a1">

<img width="814" alt="image" src="https://github.com/clean-css/clean-css-cli/assets/75518225/afe7df63-2bc8-45e1-86bd-e41fd70cfbb8">

**Comment by jakubpawlowicz** [comment]
Hey @XIYU11864 - please see https://github.com/clean-css/clean-css/blob/1ab8b7f8f5a4a40b926e740746eb1481621fada0/test/optimizer/level-2/properties/override-properties-test.js#L631 as an example.

There's no difference in your case because all properties you use are equally understandable by browsers.

However if you had "background: #f5f5f5;" followed by "background: linear-gradient(#f5f5f5, #ffffff);" in body styling, then *turning off* 'properties.merging' would leave both properties in the output file because `linear-gradient` is not understandable by some browsers so browser that don't understand it would fall back to `#f5f5f5`.

Hope I made it clear enough.

**Reply by XIYU11864** [reply]
Hi @jakubpawlowicz 
How about *turning on* 'properties.merging' in your case? I also see both properties in the output file which same as *turning off* it.
<img width="488" alt="image" src="https://github.com/clean-css/clean-css-cli/assets/75518225/e14dd6d2-1e03-4812-be53-4f5423a2fb2c">
<img width="738" alt="image" src="https://github.com/clean-css/clean-css-cli/assets/75518225/c9ff168f-da6b-4da5-86c5-0d88c9e6c9f3">

**Reply by jakubpawlowicz** [reply]
@XIYU11864 can you check what version of clean-css do you use? Or check it [here](https://clean-css.github.io/) with the latest version of clean-css?

**Reply by XIYU11864** [reply]
@jakubpawlowicz 
5.6.3. 
I think the option of 'properties.merging' doesn't work as we wish. It seems like the option is on by default and can not turn off.
